zebra-chain/src/transaction/tests/vectors.rs

```diff
@@ -391,3 +391,33 @@ fn zip244_txid() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn zip244_sighash() -> Result<()> {
+    zebra_test::init();
+
+    // TODO: we don't support Orchard deserialization yet; so test it only
```

Comment:
Can we test against the Zebra v4 sighash impl for all the v4 transactions in BLOCKS?

Reply:
I've added the test; I had to do a small refactoring. However, the test is failing... I'll investigate a bit, tomorrow

Reply:
Apparently the issue is in librustzcash. Changing it to what I think it's the expected behaviour, all tests pass for the older transactions. I've already raised the issue with the ECC team to confirm.

(The v5 is still failing, I haven't investigated it)